controller/internal/enforcer/acls/acl_test.go

```diff
@@ -71,8 +69,6 @@ func TestLookup(t *testing.T) {
 			err := a.addRule(r)
```

Comment:
why are we using a global slice of rules - if this is the only test lets bring it in here.

Reply:
fixed.